tests/web/Glpi/Inventory/Request.php

```diff
@@ -248,7 +248,7 @@ public function testAuthBasic()
             function () {
                 $this->http_client->request(
                     'POST',
-                    $this->base_uri . 'front/inventory.php',
+                    $this->base_uri . '/Inventory',
```

Comment:
Not sure, but maybe there is a double slash here that cause the test issue.
```suggestion
                    $this->base_uri . 'Inventory',
```